specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/integrationRuntime.json

```diff
@@ -0,0 +1,2837 @@
+      "post": {
+        "operationId": "IntegrationRuntimeStatus_Get",
+        "summary": "Get integration runtime status",
+        "description": "Get the integration runtime status",
+        "tags": [
+          "IntegrationRuntime"
+        ],
+        "deprecated": false,
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./workspace.json#/parameters/workspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/IntegrationRuntimeNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {
+              "$ref": "#/definitions/IntegrationRuntimeStatusResponse"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Azure Synapse service.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-examples": {
+          "Get status": {
+            "$ref": "./examples/IntegrationRuntimes_GetStatus.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "SubResource": {
+      "description": "Azure Synapse nested resource, which belongs to a factory.",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The resource identifier.",
+          "readOnly": true
+        },
+        "name": {
+          "type": "string",
+          "description": "The resource name.",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "description": "The resource type.",
+          "readOnly": true
+        },
+        "etag": {
+          "type": "string",
+          "description": "Etag identifies change in the resource.",
+          "readOnly": true
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "IntegrationRuntime": {
+      "description": "Azure Synapse nested object which serves as a compute resource for activities.",
+      "type": "object",
+      "discriminator": "type",
+      "properties": {
+        "type": {
+          "description": "Type of integration runtime.",
+          "$ref": "#/definitions/IntegrationRuntimeType"
+        },
+        "description": {
+          "description": "Integration runtime description.",
+          "type": "string"
+        }
+      },
+      "additionalProperties": {
```

Comment:
What additional properties are allowed here? Can they be modeled in swagger?

Reply:
> What additional properties are allowed here? Can they be modeled in swagger?

checked with backend team, they said it was a work around to keep the sdk compatible because backend returns more properties as time go.